pkgs/by-name/re/resnap/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  ffmpeg,
+  feh,
+  imagemagick_light,
+  lz4,
+}:
+let
+  name = "reSnap";
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = lib.strings.toLower name;
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudsftp";
+    repo = name;
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-thVyf1gTDPLQVtZKoWL7SGiWI++ICWqmF/Ar57I3WP8=";
+  };
+
+  postFixup = ''
+    substituteInPlace $out/bin/reSnap \
+      --replace-fail "\$0" ${name}
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    echo PATH="\$PATH:${
+      lib.strings.makeBinPath [
+        ffmpeg
+        feh
+        imagemagick_light
+        lz4
+      ]
+    }" > $out/bin/reSnap
+    cat ./reSnap.sh >> $out/bin/reSnap
+    chmod +x $out/bin/reSnap
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Take screnshots of your reMarkable tablet over SSH";
+    homepage = "https://github.com/cloudsftp/reSnap";
+    license = with lib.licenses; [ mit ];
+    maintainers = with lib.maintainers; [ _404wolf ];
+    mainProgram = name;
```

Changes to `meta` should not cause a rebuild, which would happen if `name` changes. We should explicitly set this.

```suggestion
    mainProgram = "reSnap";
```